### PR TITLE
Changed Nexus tutorial to add a Readiness probe

### PR DESCRIPTION
**Pull request by oatmealraisin**
Sorry, another small change.  The tutorial now adds a readiness probe, which will help with some stability.  @bparees @csrwng @openshift/team-documentation ptal?

### CODE REVIEW

File: dev_guide/app_tutorials/maven_tutorial.adoc [hunk 1]
```diff
@@ -78,6 +78,11 @@ $ oc set probe dc/nexus \
 	--failure-threshold 3 \
 	--initial-delay-seconds 30 \
 	-- echo ok
+oc set probe dc/nexus \
+	--readiness \
+	--failure-threshold 3 \
+	--initial-delay-seconds 30 \
+	--get-url=http://:8081/content/groups/public
```
Comment (bparees) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: why isn't the liveness probe the same?

Comment (csrwng) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: should they be? One is to test whether the container is live (which a simple cmd would test), vs the other one to test whether it's ready to serve content

Comment (bparees) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: it depends, but in this case if it's not serving content, something has gone wrong and you probably want it restarted, so i'd expect them to be the same.

Comment (bparees) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: i'd expect a more "typical" liveness vs readiness pair to be:
1) liveness checks that the app server is still running (listening on port 80 and accepting connections)
2) readiness checks that the app itself is available/up/initialized

i don't think checking that the container is still live serves much purpose, if the process inside the container is hung.

**Comment by csrwng** [comment]
@oatmealraisin you may want to mention about resource requirements and link to the place in the doc where we tell you how to set them.

**Comment by oatmealraisin** [comment]
@csrwng Roger, testing out steps defined [here](https://docs.openshift.org/latest/dev_guide/compute_resources.html#dev-limit-ranges) to see if it's the right document to point at.

**Comment by oatmealraisin** [comment]
@csrwng I added a mention for it at the top, ptal?

**Comment by oatmealraisin** [comment]
@bparees @csrwng ptal? Did I miss something?

**Comment by bparees** [comment]
@oatmealraisin anything you've already linked in the main doc you can remove from the references at the end.

File: dev_guide/app_tutorials/maven_tutorial.adoc (hunk 1 above)
Comment (bparees) [inline dev_guide/app_tutorials/maven_tutorial.adoc]: @oatmealraisin I didn't really see a satisfying answer on this.

**Comment by oatmealraisin** [comment]
@bparees I don't believe I have anything left to change, ptal?

**Comment by bparees** [comment]
lgtm. @openshift/team-documentation ptal/merge.

**Comment by vikram-redhat** [comment]
@bfallonf - PTAL and merge. This is a follow up to https://github.com/openshift/openshift-docs/pull/2778

**Comment by bfallonf** [comment]
Cool. Merging. I'll make a couple followup edits. Thanks, all!

**Pull request merged**